app/javascript/packs/application.js

```diff
@@ -9,6 +9,9 @@
 
 import { Application } from "stimulus"
 import { definitionsFromContext } from "stimulus/webpack-helpers"
+import * as Sentry from '@sentry/browser';
+
+Sentry.init({dsn: "https://2234c89fb9f34a1aaae5a8910141202e@o374501.ingest.sentry.io/5192625"});
```

Comment:
I think we need not to track sentry in dev env. We can do this only for prod 
```suggestion
if (process.env.NODE_ENV === 'production') {
  Sentry.init({dsn: "https://2234c89fb9f34a1aaae5a8910141202e@o374501.ingest.sentry.io/5192625"});
}
```

Reply:
Makes sense! 👍